### ISSUE TITLE
Multiple defunct keybindings in singleplayer and multiplayer but not campaign and missions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#525]
```
The following in not just copied from <http://www.hedgewars.org/node/4190>, 
because there are some small changes:

I have experienced some glitches about the keybindings in version 0.9.18 of 
Hedgewars. I created a team where I mapped all actions to keys to allow 
mouseless playing. I made sure nothing is double-assigned. Then I pressed each 
of these keys in the game to test which keybindings exactly fail. Here are the 
key bindings which fail for me:

- open weapons menu - the menu opens for a real short time (some milliseconds I 
think) and then closes immediately. Sometimes it is not visible at all (maybe 
it is so quickly gone that my eyes sometimes do, sometimes don't see it?). Also 
the camera moves to the right for some reason.
- all camera movement keys - they seem to be completely ignored
- record video - this may be my fault because I did not set up the backend 
correctly etc.

All other keys work as expected, including grenade bounce.

The effect of this bug can be reproduced both in multiplayer and singleplayer 
matches. It can't be reproduced in campaign and missions since they force the 
default key/mouse-bindings.

I have attached an example team file where everything is bound to a key.
```

Original issue reported on code.google.com by `almikes@aol.com` on 12 Feb 2013 at 12:47

Attachments:
* [Keyboard Team.hwt](https://storage.googleapis.com/google-code-attachments/hedgewars/issue-525/comment-0/Keyboard Team.hwt)

**Comment by GoogleCodeExporter** [comment]
```
Could you retest the ammo menu issue in .19-dev or in a your own build on the 
.18 branch?
Wondering if the ammo menu changes committed a while back might resolve it.

```

Original comment by `kyberneticist@gmail.com` on 12 Feb 2013 at 1:47

**Comment by GoogleCodeExporter** [comment]
```
hey mikade, isn't this the bug you mentioned on irc the other day?
```

Original comment by `vittorio...@gmail.com` on 20 Apr 2013 at 9:20

**Comment by GoogleCodeExporter** [comment]
```
Hello, it seems the bug has been fixed in 0.9.19. (I don’t know when and by 
whom, sorry.). Please close the issue. (I almost forgot the bug even existed 
:D).
```

Original comment by `almikes@aol.com` on 27 Aug 2013 at 2:01

**Comment by GoogleCodeExporter** [comment]
Original comment by `vittorio...@gmail.com` on 27 Aug 2013 at 7:24
* Changed state: **Fixed**
